core/embed/rust/src/translations/public_keys.rs

```diff
@@ -8,8 +8,8 @@ pub const PUBLIC_KEYS_DEVEL: [ed25519::PublicKey; 3] = [
 ];
 
 pub const PUBLIC_KEYS: [ed25519::PublicKey; 3] = [
-    // TODO replace with production keys
-    *b"\x00\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f",
-    *b"\x00\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f",
+    *b"\x62\xce\xa8\x25\x7b\x0b\xca\x15\xb3\x3a\x76\x40\x5a\x79\xc4\x88\x1e\xb2\x0e\xe8\x23\x46\x81\x31\x81\xa5\x02\x91\xd6\xca\xec\x67",
+    *b"\x59\x4e\xf0\x9e\x51\x13\x93\x72\xe5\x28\xc6\xc5\xb8\x74\x2e\xe1\x80\x6f\x91\x14\xca\xea\xee\xdb\x04\xbe\x6a\x98\xe1\xce\x30\x20",
+    // TODO replace with 3rd production key
```

Comment:
can we drop the 3rd one instead? (or does it fail to compile at some point)

Reply:
we can, 60a6c77c6a848b594473d9a5561e980577d01bbc